[2024-03-29T16:56:03.153+0000] {processor.py:161} INFO - Started process (PID=35) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T16:56:03.155+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T16:56:03.156+0000] {logging_mixin.py:188} INFO - [2024-03-29T16:56:03.156+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T16:56:03.163+0000] {logging_mixin.py:188} INFO - [2024-03-29T16:56:03.162+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T16:56:03.164+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T16:56:03.201+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.051 seconds
[2024-03-29T16:56:33.495+0000] {processor.py:161} INFO - Started process (PID=43) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T16:56:33.497+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T16:56:33.501+0000] {logging_mixin.py:188} INFO - [2024-03-29T16:56:33.501+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T16:56:33.510+0000] {logging_mixin.py:188} INFO - [2024-03-29T16:56:33.508+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T16:56:33.510+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T16:56:33.535+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.048 seconds
[2024-03-29T16:57:03.830+0000] {processor.py:161} INFO - Started process (PID=51) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T16:57:03.833+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T16:57:03.838+0000] {logging_mixin.py:188} INFO - [2024-03-29T16:57:03.838+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T16:57:03.845+0000] {logging_mixin.py:188} INFO - [2024-03-29T16:57:03.843+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T16:57:03.845+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T16:57:03.861+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.036 seconds
[2024-03-29T16:57:34.251+0000] {processor.py:161} INFO - Started process (PID=60) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T16:57:34.252+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T16:57:34.254+0000] {logging_mixin.py:188} INFO - [2024-03-29T16:57:34.254+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T16:57:34.259+0000] {logging_mixin.py:188} INFO - [2024-03-29T16:57:34.258+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T16:57:34.260+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T16:57:34.285+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.039 seconds
[2024-03-29T16:58:04.606+0000] {processor.py:161} INFO - Started process (PID=68) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T16:58:04.607+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T16:58:04.607+0000] {logging_mixin.py:188} INFO - [2024-03-29T16:58:04.607+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T16:58:04.612+0000] {logging_mixin.py:188} INFO - [2024-03-29T16:58:04.611+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T16:58:04.612+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T16:58:04.635+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.033 seconds
[2024-03-29T16:59:03.201+0000] {processor.py:161} INFO - Started process (PID=36) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T16:59:03.202+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T16:59:03.204+0000] {logging_mixin.py:188} INFO - [2024-03-29T16:59:03.204+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T16:59:03.207+0000] {logging_mixin.py:188} INFO - [2024-03-29T16:59:03.207+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T16:59:03.208+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T16:59:03.227+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.028 seconds
[2024-03-29T16:59:33.623+0000] {processor.py:161} INFO - Started process (PID=44) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T16:59:33.624+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T16:59:33.625+0000] {logging_mixin.py:188} INFO - [2024-03-29T16:59:33.625+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T16:59:33.628+0000] {logging_mixin.py:188} INFO - [2024-03-29T16:59:33.627+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T16:59:33.629+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T16:59:33.641+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.021 seconds
[2024-03-29T17:00:03.977+0000] {processor.py:161} INFO - Started process (PID=52) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:00:03.981+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:00:03.983+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:00:03.983+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:00:03.987+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:00:03.986+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:00:03.987+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:00:04.012+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.041 seconds
[2024-03-29T17:00:34.317+0000] {processor.py:161} INFO - Started process (PID=59) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:00:34.324+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:00:34.332+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:00:34.332+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:00:34.346+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:00:34.345+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:00:34.346+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:00:34.385+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.074 seconds
[2024-03-29T17:01:04.756+0000] {processor.py:161} INFO - Started process (PID=66) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:01:04.758+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:01:04.766+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:01:04.765+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:01:04.776+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:01:04.775+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:01:04.776+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:01:04.798+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.047 seconds
[2024-03-29T17:02:12.101+0000] {processor.py:161} INFO - Started process (PID=36) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:02:12.103+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:02:12.105+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:02:12.105+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:02:12.113+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:02:12.111+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:02:12.114+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:02:12.139+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.041 seconds
[2024-03-29T17:02:42.461+0000] {processor.py:161} INFO - Started process (PID=44) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:02:42.465+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:02:42.469+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:02:42.468+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:02:42.479+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:02:42.477+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:02:42.480+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:02:42.506+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.064 seconds
[2024-03-29T17:03:12.759+0000] {processor.py:161} INFO - Started process (PID=52) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:03:12.760+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:03:12.761+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:03:12.761+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:03:12.765+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:03:12.764+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:03:12.765+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:03:12.781+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.025 seconds
[2024-03-29T17:03:43.045+0000] {processor.py:161} INFO - Started process (PID=60) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:03:43.047+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:03:43.049+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:03:43.049+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:03:43.052+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:03:43.052+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:03:43.053+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:03:43.065+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.024 seconds
[2024-03-29T17:04:13.329+0000] {processor.py:161} INFO - Started process (PID=68) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:04:13.332+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:04:13.334+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:04:13.334+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:04:13.339+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:04:13.338+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:04:13.340+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:04:13.359+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.042 seconds
[2024-03-29T17:04:43.662+0000] {processor.py:161} INFO - Started process (PID=76) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:04:43.663+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:04:43.664+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:04:43.664+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:04:43.667+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:04:43.666+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:04:43.667+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:04:43.684+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.026 seconds
[2024-03-29T17:05:13.959+0000] {processor.py:161} INFO - Started process (PID=84) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:05:13.961+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:05:13.962+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:05:13.962+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:05:13.966+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:05:13.965+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:05:13.966+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:05:13.980+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.023 seconds
[2024-03-29T17:05:44.234+0000] {processor.py:161} INFO - Started process (PID=92) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:05:44.235+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:05:44.237+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:05:44.236+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:05:44.239+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:05:44.239+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:05:44.240+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:05:44.254+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.023 seconds
[2024-03-29T17:06:14.570+0000] {processor.py:161} INFO - Started process (PID=99) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:06:14.572+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:06:14.576+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:06:14.575+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:06:14.581+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:06:14.580+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:06:14.581+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:06:14.622+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.058 seconds
[2024-03-29T17:06:44.952+0000] {processor.py:161} INFO - Started process (PID=107) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:06:44.954+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:06:44.959+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:06:44.959+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:06:44.968+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:06:44.967+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:06:44.971+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:06:44.999+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.065 seconds
[2024-03-29T17:07:15.474+0000] {processor.py:161} INFO - Started process (PID=114) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:07:15.475+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:07:15.478+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:07:15.478+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:07:15.484+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:07:15.483+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:07:15.485+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:07:15.513+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.046 seconds
[2024-03-29T17:07:45.893+0000] {processor.py:161} INFO - Started process (PID=122) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:07:45.895+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:07:45.899+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:07:45.898+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:07:45.915+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:07:45.906+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:07:45.919+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:07:45.961+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.077 seconds
[2024-03-29T17:08:16.339+0000] {processor.py:161} INFO - Started process (PID=130) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:08:16.341+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:08:16.351+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:08:16.350+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:08:16.360+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:08:16.358+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:08:16.360+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:08:16.383+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.058 seconds
[2024-03-29T17:08:46.780+0000] {processor.py:161} INFO - Started process (PID=138) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:08:46.782+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:08:46.795+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:08:46.793+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:08:46.808+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:08:46.806+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:08:46.808+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:08:46.852+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.081 seconds
[2024-03-29T17:09:17.186+0000] {processor.py:161} INFO - Started process (PID=146) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:09:17.187+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:09:17.194+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:09:17.193+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:09:17.202+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:09:17.201+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:09:17.203+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:09:17.226+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.050 seconds
[2024-03-29T17:09:47.554+0000] {processor.py:161} INFO - Started process (PID=154) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:09:47.556+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:09:47.560+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:09:47.559+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:09:47.569+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:09:47.566+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:09:47.570+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:09:47.618+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.073 seconds
[2024-03-29T17:10:18.005+0000] {processor.py:161} INFO - Started process (PID=162) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:10:18.007+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:10:18.010+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:10:18.010+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:10:18.017+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:10:18.016+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:10:18.018+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:10:18.050+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.054 seconds
[2024-03-29T17:10:48.422+0000] {processor.py:161} INFO - Started process (PID=170) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:10:48.424+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:10:48.427+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:10:48.426+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:10:48.432+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:10:48.431+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:10:48.433+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:10:48.464+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.048 seconds
[2024-03-29T17:11:18.800+0000] {processor.py:161} INFO - Started process (PID=178) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:11:18.805+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:11:18.814+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:11:18.813+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:11:18.831+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:11:18.828+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:11:18.831+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:11:18.869+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.085 seconds
[2024-03-29T17:11:49.204+0000] {processor.py:161} INFO - Started process (PID=186) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:11:49.208+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:11:49.214+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:11:49.213+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:11:49.231+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:11:49.227+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:11:49.232+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:11:49.298+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.104 seconds
[2024-03-29T17:12:19.649+0000] {processor.py:161} INFO - Started process (PID=193) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:12:19.650+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:12:19.653+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:12:19.653+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:12:19.659+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:12:19.658+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:12:19.659+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:12:19.702+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.058 seconds
[2024-03-29T17:12:50.023+0000] {processor.py:161} INFO - Started process (PID=201) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:12:50.027+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:12:50.032+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:12:50.032+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:12:50.041+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:12:50.040+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:12:50.041+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:12:50.063+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.050 seconds
[2024-03-29T17:13:20.378+0000] {processor.py:161} INFO - Started process (PID=209) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:13:20.380+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:13:20.385+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:13:20.385+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:13:20.391+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:13:20.390+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:13:20.391+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:13:20.415+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.050 seconds
[2024-03-29T17:13:50.728+0000] {processor.py:161} INFO - Started process (PID=217) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:13:50.731+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:13:50.738+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:13:50.737+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:13:50.744+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:13:50.743+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:13:50.745+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:13:50.772+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.059 seconds
[2024-03-29T17:14:21.107+0000] {processor.py:161} INFO - Started process (PID=225) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:14:21.108+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:14:21.111+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:14:21.110+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:14:21.115+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:14:21.114+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:14:21.116+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:14:21.154+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.053 seconds
[2024-03-29T17:14:51.461+0000] {processor.py:161} INFO - Started process (PID=233) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:14:51.464+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:14:51.468+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:14:51.467+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:14:51.480+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:14:51.478+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:14:51.481+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:14:51.505+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.051 seconds
[2024-03-29T17:15:21.837+0000] {processor.py:161} INFO - Started process (PID=241) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:15:21.839+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:15:21.841+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:15:21.841+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:15:21.854+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:15:21.850+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:15:21.856+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:15:21.885+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.053 seconds
[2024-03-29T17:15:52.196+0000] {processor.py:161} INFO - Started process (PID=249) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:15:52.198+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:15:52.201+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:15:52.200+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:15:52.208+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:15:52.206+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:15:52.208+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:15:52.240+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.051 seconds
[2024-03-29T17:16:22.532+0000] {processor.py:161} INFO - Started process (PID=257) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:16:22.534+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:16:22.542+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:16:22.540+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:16:22.564+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:16:22.561+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:16:22.565+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:16:22.593+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.068 seconds
[2024-03-29T17:16:52.877+0000] {processor.py:161} INFO - Started process (PID=264) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:16:52.878+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:16:52.880+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:16:52.880+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:16:52.883+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:16:52.883+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:16:52.884+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:16:52.899+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.026 seconds
[2024-03-29T17:17:23.265+0000] {processor.py:161} INFO - Started process (PID=272) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:17:23.267+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:17:23.271+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:17:23.271+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:17:23.278+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:17:23.277+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:17:23.279+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:17:23.301+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.044 seconds
[2024-03-29T17:17:53.599+0000] {processor.py:161} INFO - Started process (PID=280) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:17:53.600+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:17:53.601+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:17:53.601+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:17:53.605+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:17:53.604+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:17:53.605+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:17:53.622+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.027 seconds
[2024-03-29T17:18:23.985+0000] {processor.py:161} INFO - Started process (PID=288) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:18:23.986+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:18:23.989+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:18:23.989+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:18:24.004+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:18:23.999+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:18:24.005+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:18:24.043+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.064 seconds
[2024-03-29T17:18:54.417+0000] {processor.py:161} INFO - Started process (PID=296) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:18:54.419+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:18:54.421+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:18:54.420+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:18:54.425+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:18:54.424+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:18:54.426+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:18:54.443+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.029 seconds
[2024-03-29T17:19:24.760+0000] {processor.py:161} INFO - Started process (PID=304) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:19:24.763+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:19:24.771+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:19:24.770+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:19:24.783+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:19:24.781+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:19:24.784+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:19:24.810+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.057 seconds
[2024-03-29T17:19:55.180+0000] {processor.py:161} INFO - Started process (PID=312) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:19:55.183+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:19:55.187+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:19:55.186+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:19:55.196+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:19:55.193+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:19:55.198+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:19:55.272+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.098 seconds
[2024-03-29T17:20:25.555+0000] {processor.py:161} INFO - Started process (PID=320) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:20:25.557+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:20:25.562+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:20:25.561+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:20:25.571+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:20:25.569+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:20:25.572+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:20:25.614+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.065 seconds
[2024-03-29T17:20:55.930+0000] {processor.py:161} INFO - Started process (PID=328) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:20:55.932+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:20:55.935+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:20:55.934+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:20:55.941+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:20:55.940+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:20:55.942+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:20:55.981+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.057 seconds
[2024-03-29T17:21:26.379+0000] {processor.py:161} INFO - Started process (PID=336) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:21:26.382+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:21:26.388+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:21:26.387+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:21:26.395+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:21:26.393+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:21:26.396+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:21:26.422+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.059 seconds
[2024-03-29T17:21:56.777+0000] {processor.py:161} INFO - Started process (PID=344) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:21:56.779+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:21:56.781+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:21:56.781+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:21:56.795+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:21:56.790+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:21:56.796+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:21:56.824+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.053 seconds
[2024-03-29T17:22:27.210+0000] {processor.py:161} INFO - Started process (PID=352) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:22:27.212+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:22:27.219+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:22:27.218+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:22:27.227+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:22:27.226+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:22:27.228+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:22:27.255+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.063 seconds
[2024-03-29T17:22:57.508+0000] {processor.py:161} INFO - Started process (PID=360) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:22:57.517+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:22:57.533+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:22:57.532+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:22:57.544+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:22:57.543+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:22:57.545+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:22:57.568+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.071 seconds
[2024-03-29T17:23:27.838+0000] {processor.py:161} INFO - Started process (PID=368) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:23:27.840+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:23:27.844+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:23:27.844+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:23:27.851+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:23:27.850+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:23:27.852+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:23:27.887+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.067 seconds
[2024-03-29T17:23:58.133+0000] {processor.py:161} INFO - Started process (PID=376) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:23:58.135+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:23:58.137+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:23:58.137+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:23:58.142+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:23:58.141+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:23:58.142+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:23:58.165+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.038 seconds
[2024-03-29T17:24:28.424+0000] {processor.py:161} INFO - Started process (PID=384) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:24:28.426+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:24:28.429+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:24:28.428+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:24:28.433+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:24:28.432+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:24:28.433+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:24:28.453+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.034 seconds
[2024-03-29T17:24:58.871+0000] {processor.py:161} INFO - Started process (PID=392) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:24:58.874+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:24:58.890+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:24:58.885+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:24:58.907+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:24:58.905+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:24:58.907+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:24:58.927+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.066 seconds
[2024-03-29T17:25:29.271+0000] {processor.py:161} INFO - Started process (PID=400) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:25:29.273+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:25:29.275+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:25:29.275+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:25:29.280+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:25:29.278+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:25:29.280+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:25:29.300+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.034 seconds
[2024-03-29T17:25:59.654+0000] {processor.py:161} INFO - Started process (PID=408) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:25:59.656+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:25:59.660+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:25:59.660+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:25:59.664+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:25:59.663+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:25:59.665+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:25:59.685+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.036 seconds
[2024-03-29T17:26:30.024+0000] {processor.py:161} INFO - Started process (PID=416) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:26:30.026+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:26:30.028+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:26:30.028+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:26:30.033+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:26:30.032+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:26:30.033+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:26:30.053+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.034 seconds
[2024-03-29T17:27:00.376+0000] {processor.py:161} INFO - Started process (PID=424) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:27:00.378+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:27:00.380+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:27:00.380+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:27:00.385+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:27:00.384+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:27:00.385+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:27:00.407+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.038 seconds
[2024-03-29T17:27:30.733+0000] {processor.py:161} INFO - Started process (PID=432) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:27:30.734+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:27:30.737+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:27:30.736+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:27:30.743+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:27:30.742+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:27:30.744+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:27:30.762+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.034 seconds
[2024-03-29T17:28:01.135+0000] {processor.py:161} INFO - Started process (PID=440) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:28:01.139+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:28:01.148+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:28:01.148+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:28:01.158+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:28:01.156+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:28:01.159+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:28:01.180+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.054 seconds
[2024-03-29T17:28:31.583+0000] {processor.py:161} INFO - Started process (PID=448) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:28:31.587+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:28:31.590+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:28:31.590+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:28:31.595+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:28:31.594+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:28:31.596+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:28:31.617+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.052 seconds
[2024-03-29T17:29:02.034+0000] {processor.py:161} INFO - Started process (PID=456) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:29:02.038+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:29:02.043+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:29:02.042+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:29:02.049+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:29:02.048+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:29:02.050+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:29:02.070+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.052 seconds
[2024-03-29T17:29:32.394+0000] {processor.py:161} INFO - Started process (PID=464) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:29:32.396+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:29:32.399+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:29:32.398+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:29:32.404+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:29:32.402+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:29:32.405+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:29:32.424+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.037 seconds
[2024-03-29T17:30:02.785+0000] {processor.py:161} INFO - Started process (PID=472) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:30:02.787+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:30:02.789+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:30:02.789+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:30:02.795+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:30:02.794+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:30:02.795+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:30:02.822+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.042 seconds
[2024-03-29T17:30:33.188+0000] {processor.py:161} INFO - Started process (PID=480) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:30:33.189+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:30:33.191+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:30:33.191+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:30:33.194+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:30:33.194+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:30:33.195+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:30:33.209+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.024 seconds
[2024-03-29T17:31:03.587+0000] {processor.py:161} INFO - Started process (PID=488) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:31:03.589+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:31:03.591+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:31:03.591+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:31:03.597+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:31:03.596+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:31:03.598+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:31:03.637+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.058 seconds
[2024-03-29T17:31:34.019+0000] {processor.py:161} INFO - Started process (PID=496) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:31:34.020+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:31:34.023+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:31:34.023+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:31:34.029+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:31:34.027+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:31:34.029+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:31:34.047+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.035 seconds
[2024-03-29T17:32:04.424+0000] {processor.py:161} INFO - Started process (PID=504) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:32:04.426+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:32:04.441+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:32:04.440+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:32:04.454+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:32:04.453+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:32:04.455+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:32:04.476+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.059 seconds
[2024-03-29T17:32:34.915+0000] {processor.py:161} INFO - Started process (PID=512) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:32:34.916+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:32:34.918+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:32:34.918+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:32:34.922+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:32:34.922+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:32:34.923+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:32:34.941+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.031 seconds
[2024-03-29T17:33:05.317+0000] {processor.py:161} INFO - Started process (PID=520) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:33:05.318+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:33:05.321+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:33:05.320+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:33:05.325+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:33:05.323+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:33:05.325+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:33:05.343+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.031 seconds
[2024-03-29T17:33:35.685+0000] {processor.py:161} INFO - Started process (PID=528) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:33:35.686+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:33:35.690+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:33:35.689+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:33:35.695+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:33:35.694+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:33:35.697+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:33:35.721+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.039 seconds
[2024-03-29T17:34:06.090+0000] {processor.py:161} INFO - Started process (PID=536) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:34:06.091+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:34:06.094+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:34:06.094+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:34:06.100+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:34:06.099+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:34:06.101+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:34:06.138+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.052 seconds
[2024-03-29T17:34:36.531+0000] {processor.py:161} INFO - Started process (PID=544) to work on /opt/airflow/dags/master_dag.py
[2024-03-29T17:34:36.532+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2024-03-29T17:34:36.534+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:34:36.534+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2024-03-29T17:34:36.537+0000] {logging_mixin.py:188} INFO - [2024-03-29T17:34:36.536+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 44
    dag=dag
    ^
SyntaxError: invalid syntax
[2024-03-29T17:34:36.537+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2024-03-29T17:34:36.559+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/master_dag.py took 0.030 seconds
